spring-boot-project/spring-boot-docs/src/docs/asciidoc/io/hazelcast.adoc

```diff
@@ -23,7 +23,7 @@ You could also specify the Hazelcast configuration file to use through configura
 	    config: "classpath:config/my-hazelcast.xml"
 ----
 
-Otherwise, Spring Boot tries to find the Hazelcast configuration from the default locations: `hazelcast.xml` in the working directory or at the root of the classpath, or a `.yaml`/`.yml` counterpart in the same locations.
+Otherwise, Spring Boot tries to find the Hazelcast configuration from the default locations: `hazelcast.xml` in the working directory or at the root of the classpath, or a `.yaml` counterpart in the same locations.
```

Comment:
I would change this to "or a YAML counterpart" rather than repeating the file extension(s).